### PR TITLE
Add cloudshell cookbook docs, and other misc update

### PR DESCRIPTION
**Pull request by jonchenn**
- Adding CloudShell cookbook 
- Fixed skaffold command in both gke and cloudrun.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -199,7 +166,7 @@ You can find more documentations in [docs](docs) folder. In a nutshell, it cover
 - [DEVELOPMENT.md](docs/DEVELOPMENT.md) - Development guide and code submission process.
 - [TROUBLESHOOTING.md](docs/TROUBLESHOOTING.md) - Development guide and code submission process.
 
-In the [docs/components](docs/components/) folder, it contains a few more guidance based on each component/feature available in this template.
+In the [docs/components](docs/components/) folder, it contains a wfew more guidance based on each component/feature available in this template.
```
Comment (lavjain) [inline README.md]: typo above

**Comment by lavjain** [comment]
Default skaffold version in cloud shell is 2.2.0
```
Welcome to Cloud Shell! Type "help" to get started.
Your Cloud Platform project in this session is set to xxx
Use “gcloud config set project [PROJECT_ID]” to change to a different project.
gcp ~ $ skaffold version
v2.2.0
gcp ~ $ 
```

**Comment by lavjain** [comment]
Also the hyperlink in Step 3.4 of `INSTALLATION.md` is broken (just fyi, not part of current PR)

**Comment by jonchenn** [comment]
> Default skaffold version in cloud shell is 2.2.0
> 
> ```
> Welcome to Cloud Shell! Type "help" to get started.
> Your Cloud Platform project in this session is set to xxx
> Use “gcloud config set project [PROJECT_ID]” to change to a different project.
> gcp ~ $ skaffold version
> v2.2.0
> gcp ~ $ 
> ```

Added a step to verify the version.

**Comment by jonchenn** [comment]
> Also the hyperlink in Step 3.4 of `INSTALLATION.md` is broken (just fyi, not part of current PR)

Fixed.

**Review by lavjain — APPROVED** [review]

**Pull request merged**